cfgov/unprocessed/css/organisms/mega-menu.less

```diff
@@ -414,27 +435,50 @@ body {
                 padding-bottom: unit( @grid_gutter-width / 2 / @base-font-size-px, em );
             }
 
+            &-link,
+            &-overview-link {
+                position: relative;
+                left: unit( -15px / 18px, em );
+                padding-top: unit( 15px / 18px, em );
+                padding-bottom: unit( 15px / 18px, em );
+                padding-left: unit( 10px / 18px, em );
+
+                border-left: 5px solid transparent;
+
+                &:hover {
+                    color: @black;
+                    border-left-color: @green;
+                }
+
+                &:focus {
+                    border-color: @black;
+                    outline: 1px dotted @black;
+                }
+
+                // This is styling for the currently selected content-2 links.
+                &__current:hover,
+                &__current,
+                &__current:visited {
+                    color: @black;
+                    border-left-color: @black;
+                    cursor: default;
```

Comment:
Why is this necessary?

Reply:
It's assuming the current page's link in the menu would not be treated as a link, so it's changing it from the hand to the default pointer cursor.